### PR TITLE
chore: add missing error check in newOutOfClusterSwfClient

### PR DESCRIPTION
**Pull request by NikeNano**
**Description of your changes:**

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: backend/src/apiserver/client/swf.go [hunk 1]
```diff
@@ -89,5 +89,9 @@ func newOutOfClusterSwfClient() (*SwfClient, error) {
 
 	// create the clientset
 	swfClientSet, err := swfclient.NewForConfig(config)
+	if err != nil {
+		return nil, err
```
Comment (capri-xiyue) [inline backend/src/apiserver/client/swf.go:93]: can you add some prefix error message like "failed to create the client set"

Comment (Bobgy) [reply backend/src/apiserver/client/swf.go:93]: nit: correct me if I'm wrong, swfclient.NewForConfig's error message should already have similar information as "failed to create the client set", therefore I think that's unnecessary.

Comment (capri-xiyue) [reply backend/src/apiserver/client/swf.go:93]: This is the source code. https://github.com/kubeflow/pipelines/blob/b482ba83d8edf8e683f315bfcf3f700970b23129/backend/src/crd/pkg/client/clientset/versioned/clientset.go#L55. I think the error message inside may not be distinct when comparing to other error messages of creating client set

Comment (NikeNano) [reply backend/src/apiserver/client/swf.go:93]: I will wrap the error to make it more clear.

**Comment by NikeNano** [comment]
@capri-xiyue and @Bobgy could you take a look again?

**Comment by Bobgy** [comment]
/lgtm
/assign @capri-xiyue 
Can you do the final approval?

**Pull request merged**